src/backend/distributed/commands/view.c

```diff
@@ -221,6 +221,33 @@ PreprocessDropViewStmt(Node *node, const char *queryString, ProcessUtilityContex
 }
 
 
+/*
+ * DropViewStmtObjectAddress returns list of object addresses in the drop view
+ * statement.
+ */
+List *
+DropViewStmtObjectAddress(Node *stmt, bool missing_ok)
+{
+	DropStmt *dropStmt = castNode(DropStmt, stmt);
+
+	List *objectAddresses = NIL;
+
+	List *possiblyQualifiedViewName = NULL;
+	foreach_ptr(possiblyQualifiedViewName, dropStmt->objects)
+	{
+		RangeVar *viewRangeVar = makeRangeVarFromNameList(possiblyQualifiedViewName);
+		Oid viewOid = RangeVarGetRelid(viewRangeVar, AccessExclusiveLock,
```

Comment:
I'd go with `AccessShareLock` here, similar to PreprocessDropTableStmt and others